packages/terra-content-container/tests/nightwatch/content-container-spec.js

```diff
@@ -9,20 +9,20 @@ module.exports = {
   },
 
   afterEach: (browser, done) => {
-    screenshot(browser, 'terra-content-container', done);
+    screenshot(browser, '#content-container', done);
   },
 
   'Displays a content container with default props': (browser) => {
     browser.url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/content-container-tests/default`);
-    browser.expect.element('.terra-ContentContainer').to.be.present;
-    browser.expect.element('.terra-ContentContainer--filler').to.not.be.present;
-    browser.expect.element('.terra-ContentContainer .terra-ContentContainer-header').to.be.present;
-    browser.expect.element('.terra-ContentContainer .terra-ContentContainer-main').to.be.present;
-    browser.expect.element('.terra-ContentContainer .terra-ContentContainer-main-normalizer').to.be.present;
+    browser.expect.element('#content-container').to.be.present;
+    browser.expect.element('#content-container').to.have.attribute('class').which.not.contains('content-container-fill');
+    browser.expect.element('#content-container div:nth-child(1)').text.to.contain('header');
+    browser.expect.element('#content-container div:nth-child(2)').text.to.contain('content');
   },
+
   'Displays a content container with fill': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/content-container-tests/fill`)
-      .assert.cssClassPresent('.terra-ContentContainer', 'terra-ContentContainer--fill');
+      .expect.element('#content-container').to.have.attribute('class').which.contains('content-container-fill');
```

Comment:
Based on our current webpack config, the nightwatch test passed. The final class name: `_-terra-content-container-lib-___ContentContainer__content-container-fill___11cbx`.
Comparing with name: `[path]___[name]__[local]___[hash:base64:5]`
```
path:_-terra-content-container-lib-
name: ContentContainer
local: content-container-fill
hash: 11cbx
```